### ISSUE TITLE
regression of testsuite in -1.2.4 under py2.7

### ISSUE THREAD

**Issue by idella** [paver/paver#143]
``` bash
PYTHON_TARGETS=python2_7 ebuild paver-1.2.3.ebuild clean test

Ran 97 tests in 0.266s

OK (SKIP=2)
 * python2_7: running distutils-r1_run_phase _clean_egg_info
>>> Completed testing dev-python/paver-1.2.3
```

``` bash
$ PYTHON_TARGETS=python2_7 ebuild paver-1.2.4.ebuild clean test

 * python2_7: running distutils-r1_run_phase python_test
........SSE................EE....................................................
======================================================================
ERROR: Failure: TypeError (patch() takes at least 2 arguments (1 given))
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/nose/loader.py", line 414, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/lib64/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/lib64/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/mnt/gen2/TmpDir/portage/dev-python/paver-1.2.4/work/Paver-1.2.4/paver/tests/test_git.py", line 6, in <module>
    @patch('paver.git.sh')
TypeError: patch() takes at least 2 arguments (1 given)

======================================================================
ERROR: Failure: TypeError (patch() takes at least 2 arguments (1 given))
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/nose/loader.py", line 414, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/lib64/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/lib64/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/mnt/gen2/TmpDir/portage/dev-python/paver-1.2.4/work/Paver-1.2.4/paver/tests/test_shell.py", line 7, in <module>
    @patch('subprocess.Popen')
TypeError: patch() takes at least 2 arguments (1 given)

======================================================================
ERROR: Failure: TypeError (patch() takes at least 2 arguments (1 given))
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/nose/loader.py", line 414, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/lib64/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/lib64/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/mnt/gen2/TmpDir/portage/dev-python/paver-1.2.4/work/Paver-1.2.4/paver/tests/test_svn.py", line 4, in <module>
    @patch('paver.svn.sh')
TypeError: patch() takes at least 2 arguments (1 given)

----------------------------------------------------------------------
Ran 81 tests in 0.289s

FAILED (SKIP=2, errors=3)
```

``` bash
$ PYTHON_TARGETS=python3_3 ebuild paver-1.2.4.ebuild clean test

 * python3_3: running distutils-r1_run_phase python_test
........SS.......................................................................................
----------------------------------------------------------------------
Ran 97 tests in 0.456s

OK (SKIP=2)
 * python3_3: running distutils-r1_run_phase _clean_egg_info
>>> Completed testing dev-python/paver-1.2.4
```

The fail in py2.7 also cause reduction of ~15 tests.  All else are equal. They all pertain to the @patch() function or decorator.
Do you require anything further?

**Comment by Arfrever** [comment]
This problem is caused by `*.pyc` files present in Paver-1.2.4.tar.gz. (They are only in `paver/tests` subdirectory.)

**Comment by Almad** [comment]
Ohh, bummer. Thanks for the report!

**Comment by Almad** [comment]
Fixed in 15ab41f35a695d4c78d8ebdb7e4c6d1c82943026
